ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -2695,7 +2683,7 @@ FEDataManager::collectActivePatchElements(std::vector<std::vector<Elem*> >& acti
             const double* const patch_dx = patch_geom->getDx();
             const double patch_dx_min = *std::min_element(patch_dx, patch_dx + NDIM);
 
-            std::set<Elem*>::const_iterator el_it = frontier_elems.begin();
+            auto el_it = frontier_elems.begin();
             const std::set<Elem*>::const_iterator el_end = frontier_elems.end();
             for (; el_it != el_end; ++el_it)
```

Comment:
I picked up this kind of construct from the libMesh examples. It seems like an ideal use case for range-based for. The tools did not do the conversion since the ranges are set outside of the `for` loop. :-(